### PR TITLE
Fix bsc#1183651: Replace pingd RA -> ping

### PR DESCRIPTION
**Pull request by tomschr**
### Description

Deprecate "pingd" resource agent in favor of "ping"

@gao-yan Yan, this is a naive implementation (just replaced `pingd` -> `ping`). Probably this isn't enough. The glossary item is disabled and won't be shown in the output.

Would you mind to have a quick look? :slightly_smiling_face: 

### Backports

- [ ] To maintenance/SLEHA15SP2
- [ ] To maintenance/SLEHA15SP1
- [ ] To maintenance/SLEHA15
- [ ] To maintenance/SLEHA12SP5
- [ ] To maintenance/SLEHA12SP4

### CODE REVIEW

File: xml/ha_config_cli.xml
Comment (gao-yan) [inline xml/ha_config_cli.xml:1281]: Probably we should point out the parameter `host_list` is a space separated list of hosts to ping and count?

I realized the primitive resource should be cloned so that each node has a running instance:

```
clone cl-ping ping \
    meta interleave=true
```

**Review by gao-yan — COMMENTED** [review]
Thanks, @tomschr.

It makes sense to have this consistent content for all supported releases.

Besides, where do you think we should probably also add a description in release note and/or in this doc to explicitly deprecate  ocf:pacemaker:pingd RA in favor of ocf:pacemaker:ping RA?

**Comment by tomschr** [comment]
@gao-yan Thanks Yan! :+1: 

> It makes sense to have this consistent content for all supported releases.

Absolutely! After I got your approval I'll cherry-pick them into the maintenance branches.

> Besides, where do you think we should probably also add a description in release note and/or in this doc to explicitly deprecate ocf:pacemaker:pingd RA in favor of ocf:pacemaker:ping RA?

Yes, that could help to put a small note into the releasenotes. Contact @KucharczykL for the details.

File: xml/ha_config_cli.xml
Comment (tomschr) [reply xml/ha_config_cli.xml:1281]: > Probably we should point out the parameter `host_list` is a space separated list of hosts to ping and count?

Good point. I've added:

    The parameter <parameter>host_list</parameter> is a space-separated list
    of hosts to ping and count.

> I realized the primitive resource should be cloned so that each node has a running instance:
> 
> ```
> clone cl-ping ping \
>     meta interleave=true
> ```

Ok, I have now this:

```
crm(live)configure# primitive ping ping \
    params name=ping dampen=5s multiplier=100 host_list="r1 r2"
crm(live)configure# clone cl-ping ping meta interleave=true
crm(live)configure# location loc-node_pref internal_www \
    rule 50: #uname eq alice \
    rule ping: defined ping
```

**Review by gao-yan — APPROVED** [review]
Looks good. Thanks!

**Pull request merged**
